src/software/ai/hl/stp/play/BUILD

```diff
@@ -358,8 +358,24 @@ cc_test(
     ],
 )
 
+py_test(
+    name = "kickoff_test",
+    srcs = [
+        "kickoff_test.py",
+    ],
```

Comment:
nit: we should add play to the names here

Reply:
fixed